flaskapi/setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup, find_packages
+
+setup(name='opgflaskapi', 
+        version='1.0', 
+        description='base flask api',
+        install_requires = ['aws_xray_sdk', 'flask-sqlalchemy', 'flask', 'psycopg2-binary', 'gunicorn'],
```

Comment:
Suggest you include version specifiers for each dependency, so we pin to at least a major version of the package, e.g.
```
install_requires = [
    'mydependency>=2,<3'
]
```

Reply:
Done